### PR TITLE
[Choreo] Changes to work with Envoy latest rate-limit image

### PR DESCRIPTION
**Pull request by slahirucd7**
### Purpose
$subject

- Envoy rate-limit image will be used replacing the existing forked rate-limit image.
- Go-control plane version update

### Issues

Fixes #

### Automation tests
 - Unit tests added: No
 - Integration tests added: YNo

### Tested environments

Not Tested

---
#### Maintainers: Check before merge
- [x] Assigned 'Type' label
- [x] Assigned the project
- [x] Validated respective github issues
- [x] Assigned milestone to the github issue(s)

### CODE REVIEW

**Comment by renuka-fernando** [comment]
Let's remove the "Do changes to the Rate Limiter service" section of the README file.
https://github.com/wso2/product-microgateway/tree/choreo/rate-limiter#do-changes-to-the-rate-limiter-service

**Comment by renuka-fernando** [comment]
If there is any file or dir in this location `product-microgateway/rate-limiter/rate-limiter-service` let's remove that too.

File: .gitmodules [hunk 1]
```diff
@@ -1,9 +1,3 @@
 [submodule "envoy-filters/envoy"]
 	path = envoy-filters/envoy
 	url = https://github.com/envoyproxy/envoy.git
```
Comment (pubudu538) [inline .gitmodules:3]: Can't we remove all the submodules?

Comment (slahirucd7) [reply .gitmodules]: fixed with: https://github.com/wso2/product-microgateway/pull/3245/commits/593f3343d74d0e8b9c8c1cc60637c28d3e5076d5

Comment (renuka-fernando) [reply .gitmodules]: If we are removing this, let's remove files in the path `<product-microgateway>/envoy-filters/envoy` as well.

**Review by pubudu538 — APPROVED** [review]

File: .gitmodules (hunk 1 above)
Comment (slahirucd7) [reply .gitmodules]: removed with : https://github.com/wso2/product-microgateway/pull/3245/commits/aa94416781242d52347b747a4942483308279e61

**Review by renuka-fernando — APPROVED** [review]

**Review by pubudu538 — APPROVED** [review]

**Comment by choreo-cicd** [comment]
[failed] Controlplane cluster : dev-deployment-v2 : 20230214.38

**Comment by choreo-cicd** [comment]
[failed]  : dev-deployment-v2 : 20230214.38

**Comment by choreo-cicd** [comment]
[failed] DataplaneE1 cluster : dev-deployment-v2 : 20230214.38

**Comment by choreo-cicd** [comment]
[succeeded] Controlplane cluster : dev-deployment-v2 : 20230214.40

**Comment by choreo-cicd** [comment]
[succeeded] DataplaneE1 cluster : dev-deployment-v2 : 20230214.40

**Comment by choreo-cicd** [comment]
[succeeded]  : dev-deployment-v2 : 20230214.40

**Comment by choreo-cicd** [comment]
[failed] Controlplane cluster : stage-deployment-v2 : 20230215.4

**Comment by choreo-cicd** [comment]
[failed] DataplaneE1 cluster : stage-deployment-v2 : 20230215.4

**Comment by choreo-cicd** [comment]
[failed]  : stage-deployment-v2 : 20230215.4

**Comment by choreo-cicd** [comment]
[failed]  : stage-deployment-v2 : 20230215.4

**Comment by choreo-cicd** [comment]
[failed] Controlplane cluster : stage-deployment-v2 : 20230215.4

**Comment by choreo-cicd** [comment]
[succeeded]  : stage-deployment-v2 : 20230215.4

**Comment by choreo-cicd** [comment]
[succeeded] Controlplane cluster : stage-deployment-v2 : 20230215.4

**Comment by choreo-cicd** [comment]
[succeeded]  : prod-deployment-v2 : 20230216.1

**Comment by choreo-cicd** [comment]
[succeeded] Controlplane cluster : prod-deployment-v2 : 20230216.1

**Pull request merged**